Resources/doc/versioning.rst

```diff
@@ -0,0 +1,180 @@
+API versioning
+==============
+Why version your API ?
+----------------------
+If you want to introduce breaking changes in your API, you should think about version your API. Indeed, if you introduce them on your current API, your users won't necessary upgrade their application directly and their applications won't work.
```

Comment:
``think about versioning your API``

``if you introduce changes to your current API, your users might not upgrade their applications right away, so versioning is important to prevent existing applications to break when such changes are made.``